charts/templates/deployment.yaml

```diff
@@ -40,17 +40,8 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.deployment.image.repository }}:{{ .Values.deployment.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: IfNotPresent
-          command:
```

Comment:
This command already exists inside [Dockerfile](https://github.com/medplum/medplum/blob/main/Dockerfile#L34), so it is preferred to not override it.

Medplum server, and therefore the Docker image, take one argument, which is "where to load config from".  Rather than trying to use YAML logic to construct that string, I think it will be better to treat that as a single opaque value that the system administrator can enter directly.